src/java/org/apache/cassandra/db/compaction/CompactionHistoryTabularData.java

```diff
@@ -74,10 +74,11 @@ public static TabularData from(UntypedResultSet resultSet) throws OpenDataExcept
             long bytesIn = row.getLong(ITEM_NAMES[4]);
             long bytesOut = row.getLong(ITEM_NAMES[5]);
             Map<Integer, Long> rowMerged = row.getMap(ITEM_NAMES[6], Int32Type.instance, LongType.instance);
-
+            String compactionType = row.getString(ITEM_NAMES[7]);
```

Comment:
if `row.getString` returns `null` here, `compactionType` should be set to `UKNOWN` here.

Reply:
ok